PurchasesCoreSwift/LocalReceiptParsing/Builders/AppleReceiptBuilder.swift

```diff
@@ -66,10 +66,10 @@ class AppleReceiptBuilder {
                 bundleId = internalContainer.internalPayload.toString()
             case .creationDate:
                 let internalContainer = try containerBuilder.build(fromPayload: payload)
-                creationDate = internalContainer.internalPayload.toDate(dateFormatter: dateFormatter)
+                creationDate = internalContainer.internalPayload.toDate()
```

Comment:
no longer injected, but we weren't really testing for the formatter to be called, rather we're testing for the end result, and we have separate tests for the formatter. 
Not ideal, but not really a downgrade from the test robustness we currently have.

Reply:
Also simplifies code in a few places.